docs/monthly-meeting/2024-11.md

```diff
@@ -23,13 +23,13 @@ to read through it!
 
 (Name / `@GitHubUsername` _[/ Discord, if different]_)
 
-- Hugo van Kemenade / @hugovk
+- Hugo van Kemenade / `@hugovk` 
```

Comment:
Lint fix:
```suggestion
- Hugo van Kemenade / `@hugovk`
```

You can use pre-commit locally to catch these things: https://pre-commit.com/#install

```
pip install pre-commit
pre-commit install
```

Then commit as normal.

Or to run for all files:
```
pre-commit run --all-files
```

And it will autofix some of the things.

It'll also help for the other PR.